cloudwatch_logs_common/src/log_publisher.cpp

```diff
@@ -112,6 +112,7 @@ void LogPublisher::Run()
 {
   Aws::CloudWatchLogs::ROSCloudWatchLogsErrors publisher_status = CW_LOGS_SUCCEEDED;
   Aws::String next_token;
+  this->cloudwatch_facade_->GetLogStreamToken(this->log_group_, this->log_stream_, next_token);
```

Comment:
Will this be ok if the log group doesn't exist yet? The if does_group_exist check is happening on line 118. Users were confused about red-herring errors being thrown about missing log groups in the past.

Reply:
This will generate an error message if the stream hasn't been created yet. I'm doing a small refactor to handle that.